draft-ietf-mls-protocol.md

```diff
@@ -2470,16 +2470,16 @@ psk_secret (or 0) --> KDF.Extract
 
 A number of secrets are derived from the epoch secret for different purposes:
 
-| Secret                  | Label           |
-|:------------------------|:----------------|
-| `sender_data_secret`    | "sender data"   |
-| `encryption_secret`     | "encryption"    |
-| `exporter_secret`       | "exporter"      |
-| `authentication_secret` | "authentication"|
-| `external_secret`       | "external"      |
-| `confirmation_key`      | "confirm"       |
-| `membership_key`        | "membership"    |
-| `resumption_secret`     | "resumption"    |
+| Label            | Secret                | Purpose                                                     |
+|:-----------------|:----------------------|:------------------------------------------------------------|
+| "sender data"    | `sender_data_secret`  | Deriving keys to encrypt sender data                        |
+| "encryption"     | `encryption_secret`   | Deriving message encryption keys (via the secret tree)      |
+| "exporter"       | `exporter_secret`     | Deriving exported secrets                                   |
+| "external"       | `external_secret`     | Deriving the external init key                              |
+| "confirm"        | `confirmation_key`    | Computing the confirmation MAC for an epoch                 |
+| "membership"     | `membership_key`      | Computing the membership MAC for an MLSPlaintext            |
+| "resumption"     | `resumption_psk`      | Proving membership in a prior epoch (via a PSK)             |
```

Comment:
Isn't it to prove the membership of the current epoch, to be used in a future epoch?